apps/dashboard-service/src/dashboard/hectares-restoration.controller.ts

```diff
@@ -21,9 +20,6 @@ export class HectaresRestorationController {
   @JsonApiResponse([HectareRestorationDto])
   @ApiOperation({ operationId: "getHectaresRestoration", summary: "Get hectares restoration" })
   async getHectaresRestoration(@Query() query: DashboardQueryDto) {
-    if (isEmpty(query.projectUuid)) {
-      throw new BadRequestException("Invalid projectUuid");
-    }
```

Comment:
I believe this should also be restored - it's an error to load this endpoint without a project ID.